app/src/main/java/com/blackboxteam/app/blackbox/util/UserSessionPersistence.java

```diff
@@ -0,0 +1,38 @@
+package com.blackboxteam.app.blackbox.util;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.widget.Toast;
+
+/**
+ * UserSessionPersistence
+ *
+ * Manages user session
+ */
+public class UserSessionPersistence {
+    private Context mContext;
+    private String user;
+    private SharedPreferences sharedPreferences;
+
+    public UserSessionPersistence(Context context) {
+        mContext = context;
+    }
+
+    // Save the user login/signIp info
+    public boolean saveUserInfo(String email, String password) {
+        sharedPreferences = mContext.getSharedPreferences("userInfo", Context.MODE_PRIVATE);
```

Comment:
To make things cleaner, we can get default sharedPreferences using: ```PreferenceManager.getDefaultSharedPreferences(application);``` We can call this once in the constructor, passing in  application.

Reply:
Getting cannot resolve symbol application

Reply:
!